packages/components/app/styles/components/table.scss

```diff
@@ -2,107 +2,173 @@
 // TABLE
 //
 //
+$hds-table-border-radius: 6px;
+$hds-table-border-width: 1px;
+$hds-table-inner-border-radius: $hds-table-border-radius - $hds-table-border-width;
+$hds-table-border-color: var(--token-color-border-primary);
+
 .hds-table,
 .hds-table-sortable {
   width: 100%;
   font-size: var(--token-typography-body-200-font-size);
   font-family: var(--token-typography-font-stack-text);
   line-height: var(--token-typography-body-200-line-height);
   table-layout: fixed;
-  border: 1px solid var(--token-color-border-primary);
-  border-radius: 6px;
-  border-collapse: collapse;
+  border: $hds-table-border-width solid $hds-table-border-color;
+  border-radius: $hds-table-border-radius;
   border-spacing: 0;
 }
 
+// TABLE HEADER
+.hds-table__thead {
+  .hds-table__tr {
+    color: var(--token-color-foreground-strong);
+    background-color: var(--token-color-surface-strong);
+
+    .hds-table__th,
+    .hds-table__th-sort {
+      height: 48px;
+      color: var(--token-color-foreground-strong);
+      font-weight: var(--token-typography-font-weight-semibold);
+      font-size: 14px;
+      line-height: 1.4;
+      text-align: left;
+      background-color: var(--token-color-surface-strong);
+      border-top: none;
+      border-right: $hds-table-border-width solid $hds-table-border-color;
+      border-bottom: $hds-table-border-width solid $hds-table-border-color;
+      border-left: none;
+    }
+
+    .hds-table__th {
+      padding: 12px 16px;
```

Comment:
thought: I see this padding quite a bit across the code (`.hds-table__th`, `.hds-table__th-sort button` and `.hds-table--density-medium .hds-table__tbody td`) and was wondering if we'd want to store these padding values (and maybe the other densities) in SCSS variables.

Reply:
That's probably a good idea, thanks for mentioning it!

Reply:
ok I've extracted the padding values to variables. Looking at other components it seems like we _also_ do some more complex loops, but since the padding is the only thing that changes (vs the button where nearly all the props change based on the size), didn't go that route.